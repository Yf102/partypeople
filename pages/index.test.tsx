import React from 'react'
import { render, screen, waitFor } from 'tests/test-utils'
import { Partners } from 'tests/__mocks__/data/Partners'
import Home from 'pages/index'

describe('HomePage', () => {
  it('It renders Home page', async () => {
    const { asFragment } = render(<Home partners={Partners} />)
    expect(screen.getByTestId('home-element')).toBeTruthy()
    expect(asFragment()).toMatchSnapshot()
  })
})
