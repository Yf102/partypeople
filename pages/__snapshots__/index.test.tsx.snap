// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage It renders Home page 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="home-element"
  >
    <div
      class="grid grid-cols-12 auto-rows-max mx-auto max-w-3xl"
    >
      <div
        class="col-span-12 px-3 md:px-6"
      >
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            1
          </div>
          <h1
            class="text-xl"
          >
            Jamelia Waller
          </h1>
          <div
            class="paragraph mb-0"
          >
            11.732 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            3
          </div>
          <h1
            class="text-xl"
          >
            Gracie-Leigh Mccallum
          </h1>
          <div
            class="paragraph mb-0"
          >
            6.513 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            5
          </div>
          <h1
            class="text-xl"
          >
            Margot Mckay
          </h1>
          <div
            class="paragraph mb-0"
          >
            49.044 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            6
          </div>
          <h1
            class="text-xl"
          >
            Jodie Dyer
          </h1>
          <div
            class="paragraph mb-0"
          >
            11.776 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            8
          </div>
          <h1
            class="text-xl"
          >
            Ffion Firth
          </h1>
          <div
            class="paragraph mb-0"
          >
            5.531 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            9
          </div>
          <h1
            class="text-xl"
          >
            Billie-Jo Dunlap
          </h1>
          <div
            class="paragraph mb-0"
          >
            6.846 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            10
          </div>
          <h1
            class="text-xl"
          >
            Sachin Chan
          </h1>
          <div
            class="paragraph mb-0"
          >
            10.88 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            11
          </div>
          <h1
            class="text-xl"
          >
            Una Beach
          </h1>
          <div
            class="paragraph mb-0"
          >
            5.191 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            12
          </div>
          <h1
            class="text-xl"
          >
            Bluebell Robles
          </h1>
          <div
            class="paragraph mb-0"
          >
            4.766 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            13
          </div>
          <h1
            class="text-xl"
          >
            Nur Combs
          </h1>
          <div
            class="paragraph mb-0"
          >
            64.566 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            14
          </div>
          <h1
            class="text-xl"
          >
            Vincenzo Conner
          </h1>
          <div
            class="paragraph mb-0"
          >
            15.541 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            16
          </div>
          <h1
            class="text-xl"
          >
            Maddy Novak
          </h1>
          <div
            class="paragraph mb-0"
          >
            3.222 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            17
          </div>
          <h1
            class="text-xl"
          >
            Kodi Collier
          </h1>
          <div
            class="paragraph mb-0"
          >
            4.781 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            18
          </div>
          <h1
            class="text-xl"
          >
            Vladimir Vasov
          </h1>
          <div
            class="paragraph mb-0"
          >
            48.79 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            19
          </div>
          <h1
            class="text-xl"
          >
            Butterfree Crackersprout
          </h1>
          <div
            class="paragraph mb-0"
          >
            11.922 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            20
          </div>
          <h1
            class="text-xl"
          >
            Oscarbait Crackersprout
          </h1>
          <div
            class="paragraph mb-0"
          >
            9.879 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            21
          </div>
          <h1
            class="text-xl"
          >
            Bendydick Chickenstrips
          </h1>
          <div
            class="paragraph mb-0"
          >
            17.257 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            24
          </div>
          <h1
            class="text-xl"
          >
            Rada Ilieva
          </h1>
          <div
            class="paragraph mb-0"
          >
            16.884 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            25
          </div>
          <h1
            class="text-xl"
          >
            Bombadil Rivendell
          </h1>
          <div
            class="paragraph mb-0"
          >
            9.22 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            27
          </div>
          <h1
            class="text-xl"
          >
            Francisco Fleming
          </h1>
          <div
            class="paragraph mb-0"
          >
            25.46 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            28
          </div>
          <h1
            class="text-xl"
          >
            Romy Harrison
          </h1>
          <div
            class="paragraph mb-0"
          >
            12.497 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            30
          </div>
          <h1
            class="text-xl"
          >
            Bunsenburner Crumplehorn
          </h1>
          <div
            class="paragraph mb-0"
          >
            6.354 km
          </div>
        </div>
        <div
          class="light col-span-12 col-start-1 text-center block rounded-lg px-8 py-10 mb-4"
          data-testid="partner-card-element"
        >
          <img
            alt=""
            class="opacity-30"
            data-nimg="future"
            decoding="async"
            height="42"
            src="/icons/icon_user.svg"
            srcset="/icons/icon_user.svg 1x, /icons/icon_user.svg 2x"
            style="color: transparent;"
            width="42"
          />
          <div
            class="paragraph"
          >
            39
          </div>
          <h1
            class="text-xl"
          >
            Patience Lyon
          </h1>
          <div
            class="paragraph mb-0"
          >
            10.428 km
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
